source/get-started/cloud-install/import-clr-aws.rst

```diff
@@ -0,0 +1,509 @@
+      :scale: 100%
+      :alt: AWS Services - Snapshots
+
+      Figure 23: AWS Services - Snapshots
+
+  #. Locate the snaphot using the `Snapshot ID`.
+     See Figure 24.
+
+   .. figure:: ../../_figures/aws/import-clr-aws-24.png
+      :scale: 100%
+      :alt: AWS Services - Snapshots
+
+      Figure 24: AWS Services - Snapshots
+    
+  #. Right-click it and select :guilabel:`Create Image`.
+
+  #. Configure as follows:
+ 
+     * Enter the name in the :guilabel:`Name` field
+     * Enter the description in the :guilabel:`Description` field
+     * Set the :guilabel:`Architecture` as `x86_64`
+     * Set the :guilabel:`Virtualization type` as `Hardware-assisted virtualization`
+     * Set the :guilabel:`Root device name` as `/dev/sda1`
+
+     See Figure 25.
+
+     .. figure:: ../../_figures/aws/import-clr-aws-25.png
+        :scale: 100%
+        :alt: AWS Services - Snapshots
+
+        Figure 25: AWS Services - Snapshots
+    
+  #. Click :guilabel:`Create`.
+
+Launch an instance
+******************
+
+#. Go to :guilabel:`Services`, :guilabel:`Compute`, and select 
+   :guilabel:`EC2`.
+   See Figure 26. 
+
+   .. figure:: ../../_figures/aws/import-clr-aws-26.png
+      :scale: 100%
+      :alt: AWS Services - EC2
+
+      Figure 26: AWS Services - EC2
+    
+#. Click the :guilabel:`Launch Instance` dropdown and select 
+   :guilabel:`Launch Instance`.
+   See Figure 27. 
+
+   .. figure:: ../../_figures/aws/import-clr-aws-27.png
+      :scale: 100%
+      :alt: AWS Services - Launch instance
+
+      Figure 27: AWS Services - Launch instance
+
+#. On the left navigation bar, select :guilabel:`My AMIs`.
+   See Figure 28. 
+
+   .. figure:: ../../_figures/aws/import-clr-aws-28.png
+      :scale: 100%
+      :alt: AWS Services - Select AMI
+
+      Figure 28: AWS Services - Select AMI
+
+#. Find your AMI and click :guilabel:`Select`.
+
+#. From here onward, configure the details of your instance as desired 
+   and launch it.
+   
+Connect to your |CL| instance
+*****************************
+
+#. Follow these steps to `connect to your instance`_.
+
+Related topics
+**************
+
+* :ref:`azure`
+* :ref:`gce`
+* :ref:`clr-digitalocean`
+
+.. _Downloads: 
+   https://clearlinux.org/downloads
+.. _aws.yaml: 
+   https://cdn.download.clearlinux.org/current/config/image/aws.yaml
+.. _AWS image post-install script:
+   https://cdn.download.clearlinux.org/current/config/image/aws-disable-root.sh
+.. _Installing the AWS CLI version 2: 
+   https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html
+.. _Configuring the AWS CLI: 
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+.. _connect to your instance: 
+   https://docs.01.org/clearlinux/latest/get-started/cloud-install/aws-web.html#connect-to-your-clear-linux-os-basic-instance
+.. _Installer YAML Syntax:
+   https://github.com/clearlinux/clr-installer/blob/master/scripts/InstallerYAMLSyntax.md
+
```

Comment:
Looks like you just need to add a hyperlink for `post-install`_ here -- to pass build test.

Reply:
done!